.gitignore

```diff
@@ -0,0 +1,6 @@
+# Default ignored files
+/shelf/
```

Comment:
please remove this file.
I found another way to fix your issue with foolders which should be ignored for commit.
Please update your branch, 
pull last changes from master or do rebase.